components/suggest/src/store.rs

```diff
@@ -667,13 +673,21 @@ where
             Err(_) => Ok(()),
         }
     }
+
+    fn should_reingest_record(&self, dao: &mut SuggestDao, record: &Record) -> Result<bool> {
+        match &record.payload {
+            SuggestRecord::Exposure(_) => Ok(!dao.is_exposure_suggestion_ingested(&record.id)?),
```

Comment:
Can you add a comment explaining that this can happen when the constraints change?